deploy.sh

```diff
@@ -2,23 +2,26 @@
 
 # Install dependencies
 yarn;
-yarn global add mup https://github.com/jimrandomh/mup-aws-beanstalk;
+yarn global add mup@1.4.6 https://github.com/jimrandomh/mup-aws-beanstalk;
 yarn global add json;
 yarn global add @sentry/cli;
 
 # Variables & Arguments
 settings_path="$1";
 deploy_config_path="$2";
-sentry_version=$(sentry-cli releases propose-version);
+echo 'deploy_config_path'
+# sentry_version=$(sentry-cli releases propose-version);
 
+git clone https://github.com/elasticdog/transcrypt.git
 # Environment variables and secrets provided via travis-ci
-git clone https://Discordius:$GITHUB_TOKEN@github.com/LessWrong2/LessWrong-Credentials.git
+./transcrypt/transcrypt -c aes-256-cbc -p $TRANSCRYPT_SECRET -y
 
-# Set up sentry configurations
-sentry-cli releases new -p lesswrong $sentry_version;
-sentry-cli releases set-commits --auto $sentry_version;
-json -I -f $settings_path -e "this.public.sentry.release=\"$sentry_version\"";
+# # Set up sentry configurations
```

Comment:
Should Sentry be set up? If not, 💀

Reply:
Not yet. And although it pains me to leave commented out code, I do want to lower the diff — actually, you know what, if they change a line we've deleted that's fine. When we install Sentry, we'll just steal their code anew.